### DISCUSSION TITLE
How can I get the "Install Kit" link?

### DISCUSSION THREAD

**Discussion by manan30** [johnlindquist/kit#765] [Q&A] [answered]
Hello!

I recently published a script and was wondering how can I get the "Install Kit" link. For context here is the script that I published https://github.com/johnlindquist/kit/discussions/763. Appreciate any help. Thank you!

**Comment by johnlindquist** [comment, marked as answer]
From the main menu, press the right arrow key or click on this little arrow
![CleanShot 2022-05-02 at 15 26 43@2x](https://user-images.githubusercontent.com/36073/166330427-f377ea9a-8b86-410e-bd83-bed3f7dc436b.png)

Options will appear to share, etc:

<img width="849" alt="CleanShot 2022-05-02 at 15 26 08@2x" src="https://user-images.githubusercontent.com/36073/166330353-402fbe76-d154-430e-9b51-84e62e4c18cd.png">

The "Share as Github Discussion" should format a nice post to copy/paste

**Comment by manan30** [comment]
Perfect! Thank you so much for this detailed explanation!
